pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -155,7 +155,7 @@ func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo
 		// exclusively allocated.
 		reservedCPUsFloat := float64(reservedCPUs.MilliValue()) / 1000
 		numReservedCPUs := int(math.Ceil(reservedCPUsFloat))
-		policy, err = NewStaticPolicy(topo, numReservedCPUs, specificCPUs, affinity)
+		policy, err = NewStaticPolicy(topo, numReservedCPUs, specificCPUs, staticCPUs, affinity)
```

Comment:
I have a pending PR (https://github.com/kubernetes/kubernetes/pull/87522) which also adds a new parameter to static policy creation indicating the available CPUs (using a cpuset cgroup file)̣. These seem to be related approaches.